### DISCUSSION TITLE
Quick question about the role of redis..

### DISCUSSION THREAD

**Discussion by eugen-bondarev** [docmost/docmost#280] [Help / Questions] [answered]
Hi, I have a quick question: is redis used to store any persistent data? 

When I look at the code it looks like redis is used for a queue, which doesn't necessarily have to be persistent, right?

The reason I ask is because I'd like to create a throw-away redis server without any persistency and I wanna be sure that all the important stuff is in Postgres.

Thanks!

**Comment by Philipinho** [comment, marked as answer]
No persistent data is stored in Redis. 
It is used for queues and WebSocket data propagation across servers.

**Comment by jigarjoshi011** [comment]
Yes Redis Behaves here like a Pub/Sub so that if the two users can have the real-time data which is produced by target
